packages/pdf-worker/src/strategies/ChatTranscript.ts

```diff
@@ -6,26 +6,23 @@ import exportChatTranscript from '../templates/ChatTranscript';
 import type { ChatTranscriptData, PDFMessage } from '../templates/ChatTranscript';
 
 export class ChatTranscript implements IStrategy {
-	private isNewDay(current: PDFMessage, previous: PDFMessage | undefined, timezone: string): boolean {
-		return !previous || !moment(current.ts).tz(timezone).isSame(previous.ts, 'day');
+	private isNewDay(current: PDFMessage, previous: PDFMessage | undefined): boolean {
+		return !previous || !moment(current.ts).isSame(previous.ts, 'day');
```

Comment:
Why are you removing the timezone param?

Reply:
It was throwing an error on the storybook, but the issue was on my side, thanks for catch it :)